### DISCUSSION TITLE
Need setting that serialbattery is only Battery monitor

### DISCUSSION THREAD

**Discussion by Joeknx** [Louisvdw/dbus-serialbattery#482] [Q&A] [answered]
Hi,

if I don't need the driver but only the numbers from my JK BMS.
Wich setting do I need in Venus OS?

If I disable DVCC I get a "low battery" alarm and the system stops working.

Thanks!

**Comment by mr-manuel** [comment]
Do you tried to set the battery monitor to a device other than `Automatic` or `SerialBattery` in `Venus OS GUI -> Settings -> System setup -> Battery monitor`?

![grafik](https://user-images.githubusercontent.com/12592765/223447671-023bfb27-3b7b-4c64-a1e8-b46a6f8823c2.png)

**Comment by Joeknx** [comment]
I just tried it, but I still get the error after 1 minute or so.

**Reply by mr-manuel** [reply]
Is this a false positive or is the battery indeed low? In the JKBMS driver the alert should be triggered from the BMS. Can you please post a screenshot of the warning and cell voltages page?

**Reply by Joeknx** [reply]
It is false. The voltages are all fine!

Maybe I just leave it as it is. It works except Floating

<img width="494" alt="Bildschirm­foto 2023-03-07 um 16 18 09" src="https://user-images.githubusercontent.com/77569160/223465887-7a3f7bc3-e7c7-408b-bf08-9805faaefe3c.png">
<img width="512" alt="Bildschirm­foto 2023-03-07 um 16 17 12" src="https://user-images.githubusercontent.com/77569160/223465901-cefd60e4-5556-4879-8196-9ea4e2a13514.png">

**Reply by mr-manuel** [reply]
It seems that this alarm is comming from the MultiPlus-II and not from the SerialBattery driver, but I'm not sure.

What are your values in the VE Configure under the `Inverter` tab?

To exclude the driver try to comment out this lines https://github.com/Louisvdw/dbus-serialbattery/blob/master/etc/dbus-serialbattery/dbushelper.py#L413-L442 by adding a `#` in front of each line. Then reboot the Venus OS.

**Reply by mr-manuel** [reply]
You could also check `SerialBattery(Jkbms) -> Alarms` and `MultiPlus-II 48/5000/70-50 -> Alarm status` when the error is triggered instead of commenting out.

**Reply by Joeknx** [reply]
ok, I try that

**Comment by Joeknx** [comment]
<img width="422" alt="Bildschirm­foto 2023-02-22 um 17 48 28" src="https://user-images.githubusercontent.com/77569160/223468871-30641f71-4f17-47ea-a1b2-3a2d2ea23e04.png">

**Reply by DirkReudenbach** [reply]
if you run an ESS System these Voltages will not be used. You'll find the voltages in the ESS Assistant then

**Comment by mr-manuel** [comment]
Maybe also this can help you: https://github.com/Louisvdw/dbus-serialbattery/issues/363

**Reply by Joeknx** [reply]
Ah, thank you. This could indeed be the problem. 
I probably wait for a new release where those problems are fixed. 99% work as it is.

**Reply by mr-manuel** [reply]
Please mark this as answer, if this ok for you

**Comment by mr-manuel** [comment, marked as answer]
I have a update for you.

The battery monitor is not the same as the BMS selected for DVCC.

With Venus OS >3.00 you can select the BMS that should be used for DVCC:
![grafik](https://user-images.githubusercontent.com/12592765/232209770-4a46e04e-bb2f-4afd-b42b-f3da7e0b63d6.png)

Source: https://community.victronenergy.com/questions/199460/multiple-battery-monitor-available-only-selected-a.html?childToView=199503#comment-199503
